lib/galaxy/config/sample/galaxy.yml.sample

```diff
@@ -1288,16 +1294,14 @@ galaxy:
   # The value of this option will be resolved with respect to
   # <data_dir>.
   #interactivetools_map: interactivetools_map.sqlite
+  # Note: the following config should still be used due to lack of
+  # support of data_dir resolution in gx-it-proxy and gravity:
+  #interactivetools_map: database/interactivetools_map.sqlite
 
   # Prefix to use in the formation of the subdomain or path for
   # interactive tools
   #interactivetools_prefix: interactivetool
 
-  # Shorten the uuid portion of the subdomain or path for interactive
-  # tools. Especially useful for avoiding the need for wildcard
-  # certificates by keeping subdomain under 63 chars
-  #interactivetools_shorten_url: false
```

Comment:
Can you mention why that was removed in the interactive tools docs ?

Reply:
The whole config is removed as it is not needed anymore (all URLs are now shortened). See discussion e.g. https://github.com/galaxyproject/galaxy/pull/16795#issuecomment-1751648456 above

Reply:
yes, can you just put the gist of that in the docs.

Reply:
ah, misunderstood you. Yes I can do that. Not sure where, though, but I'll try to find a natural place

Reply:
Added a note on this in https://github.com/galaxyproject/galaxy/pull/16795/commits/25065986e12619c6bcf7414681c1f214d0d05461. Does this look ok?

Reply:
Yes, thank you!